android/lib/pages/profile_page.dart

```diff
@@ -719,12 +740,15 @@ Widget profilePictureBuilder(picture_id) {
                 return const Text(
                     "An error occured while loading profile page!");
               }
-
+              String final_string = image_output.image!.base64String;
```

Comment:
Parsing the images like these are prone to errors, as mentioned in the above comment, use `circleAvatarBuilder()`.

Reply:
This widget is not used anymore but it returns a circleAvatarBuilder() just in case